net-wireless/nitroshare/additional-files/nitroshare.rdef

```diff
@@ -0,0 +1,41 @@
+resource app_flags B_MULTIPLE_LAUNCH;
```

Comment:
Check here how to fill in the correct values: https://github.com/Begasus/haikuports/wiki/Useful-variables-for-recipes#add-icon-and-application-information

Reply:
As explained in the link you can sed the version from inside the recipe, rename the `rdef` file to `rdef.in` and feed the info with sed, saves git globber on next version.

Reply:
Updated.